packages/extension-browser/package.json

```diff
@@ -140,7 +140,7 @@
     "lint:dependencies": "node ../../scripts/lint-dependencies.js",
     "lint:md": "node ../../scripts/lint-markdown.js",
     "prebuild": "npm-run-all prebuild:*",
-    "prebuild:assets": "cpx \"./src/**/*.{html,json,png}\" dist/bundle && cpx \"./src/*.svg\" dist/bundle && cpx \"./src/browser-action/*.svg\" dist/bundle/browser-action",
+    "prebuild:assets": "copyfiles \"./src/**/*.{html,json,png}\" dist/bundle && copyfiles \"./src/*.svg\" dist/bundle && copyfiles \"./src/browser-action/*.svg\" dist/bundle/browser-action",
```

Comment:
Judging from the CI failures, this `copyfiles` command isn't quite working the same as `cpx`. We're not getting `_locales/en/messages.json` copied over.

Reply:
yep, I was working on the fix. Please, take a look to my last commit.